src/parsers/manifestjson.js

```diff
@@ -107,6 +107,13 @@ export default class ManifestJSONParser {
 
     if (error.keyword === 'required') {
       baseObject = messages.MANIFEST_FIELD_REQUIRED;
+    } else if (error.dataPath.startsWith('/permissions') &&
```

Comment:
Indeed, as all permissions are a string, as I understand it: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/permissions

> This key is an array of strings, and each string is a request for a permission.